tests/MongoDB.Driver.Tests/PipelineStageDefinitionBuilderTests.cs

```diff
@@ -124,6 +124,105 @@ public void ChangeStream_should_return_the_expected_result_when_options_isNull()
             stage.Document.Should().Be("{ $changeStream : { } }");
         }
 
+        [SkippableFact]
+        public void GraphLookup_with_incompatible_parameters_should_throw()
+        {
+            RequireServer.Check().Supports(Feature.AggregateGraphLookupStage);
+
+            var client = new MongoClient(CoreTestConfiguration.ConnectionString.ToString());
+            var employeesCollection = client.GetDatabase("test").GetCollection<BsonDocument>("employees");
+
+            var connectFromField = (FieldDefinition<BsonDocument, string[]>)"name";
```

Comment:
While the tests may be passing it's not really correct to say that a `BsonDocument` can have a field of type `string[]`. Fields of a `BsonDocument` are always of type `BsonValue`.

Refactor the tests to use real classes.

Reply:
Done.